examples/sample_client.py

```diff
@@ -2,7 +2,12 @@
 
 from uid2_client import Uid2Client
 from uid2_client import decrypt
+from uid2_client import encrypt
+from uid2_client.identity_scope import IdentityScope
 
+# this sample client will decrypt a given advertising token into raw UID2
```

Comment:
# this sample client decrypts an advertising token into a raw UID2

Reply:
fixed